### PR TITLE
homepage_for_portfolio

### PR DESCRIPTION
**Pull request by johnspeny**
- Created images folder to hold images

- icon folder for SVG icons
- index.html with proper link to css file

### CODE REVIEW

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi  @johnspeny 👋 , 

You have done an impressive job so far! 👌.  However, there are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights 🚀 
- Great implementation of the `flexbox`
- You crafted a robust README file
- Your pull request is descriptive but can use some more work
- Proper Git flow 

## Required Changes ♻️
_Check the comments under the review._


### Optional suggestions
_Moving forward please make use of present tense for commit messages and pull request description_ 
example instead of `created images` rather `Create images` .

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <div class="header__toggler">
+            <i class="header__hamburger fa fa-bars"></i>
+        </div>
+    </div>
+
+    <section class="headline">
+        <div class="headline__inner">
+        <div>
+            <h2> <span class="headline__nameOne">Hey there. I'm Okker John.</span><br />
+            <span class="headline__nameTwo">I'm a software developer</span></h2>
```
[the remark this answers is not in the document]
Comment (johnspeny) [reply index.html]: This will break my css. its best left like that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <h3 class="header__logo">johnspeny</h3>
+        <div class="header__toggler">
+            <i class="header__hamburger fa fa-bars"></i>
+        </div>
+    </div>
+
+    <section class="headline">
+        <div class="headline__inner">
+        <div>
+            <h2> <span class="headline__nameOne">Hey there. I'm Okker John.</span><br />
```
[the remark this answers is not in the document]
Comment (johnspeny) [reply index.html]: Done

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <span class="headline__nameTwo">I'm a software developer</span></h2>
+        </div>
+        <div class="headline__bodyText"><p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
+        <div class="headline__social">
+            <span><a href="#"><img src="icons/github.svg" alt="twitter" /></a></span>
+            <span><a href="#"><img src="icons/Linkedin_icon.svg" alt="twitter"/></a></span>
+            <span><a href="#"><img src="icons/twitter.svg" alt="twitter"/></a></span>        
+        </div>
```
[the remark this answers is not in the document]
Comment (johnspeny) [reply index.html]: Done

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Porfolio</title>
+</head>
+<body>
+    <div class="header">
+        <h3 class="header__logo">johnspeny</h3>
```
[the remark this answers is not in the document]
Comment (johnspeny) [reply index.html]: Done

File: style.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.header__logo {
+  color: #6070ff;
+}
```
[the remark this answers is not in the document]
Comment (johnspeny) [reply style.css]: Done

**Comment by johnspeny** [comment]
All changes made except changing span to p tags

File: index.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        <div class="header__toggler">
+            <i class="header__hamburger fa fa-bars"></i>
+        </div>
+    </header>
+
+    <section class="headline">
+        <div class="headline__inner">
+        <div>
+            <h1><span class="headline__nameOne">Hey there. I'm Okker John.</span><br />
+            <span class="headline__nameTwo">I'm a software developer</span></h1>
```
Comment (ahmetbozaci) [inline index.html:23]: - [ ] You didn't fix last review. You wrapped this section by `h1 `tag. You should use `h1` tag and delete span in this part.  You should `h1` use tag for `Hey there. I'm Okker John` and you should use `p` tag for `I'm a software developer`

File: style.css [hunk 7]
```diff
@@ -0,0 +1,73 @@
+html,
+body {
+  font-size: 20px;
+  margin: 0 20px;
+}
+
+.header {
+  display: flex;
```
Comment (ahmetbozaci) [inline style.css:8]: **OPTIONAL**
- [ ] For DRY css cam  use combine duplicate properties. 
You can create class `.d-flex { display: flex}` and use this class in html file.

- For more information about DRY principles you can check this [article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️ 

## Hello @johnspeny ✋🏼  Nice done with working on last review.However there are still some issues you should fix. Please fix them then your project will be approved.


### ♻️ Request changes ♻️
 
_**Check the comment about that  under the review.**_


### OPTIONAL SUGGESTIONS

#### These are optional 

- [ ] Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`. Please watch this [video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) and check this [article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
- [ ]  Instead of using pictures for icons you can use icons from [font-awesome](https://fontawesome.com/start). 
### Cheers and Happy coding! 💻 👏
cat on laptop
___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

**Comment by johnspeny** [comment]
@ahmetbozaci have a review and see if its well

File: index.html (hunk 6 above)
Comment (johnspeny) [reply index.html]: fixed

**Comment by johnspeny** [comment]
@AshakaE have a look at what I have so far. I cant handle rebase at the moment, it has a weird behaviour on my visual studio code cmd that I can't control yet.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**